docs/prober-and-monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Prober and monitoring
+
+**PMAP is not a official Google product.**
+
+Monitoring consists of two parts: prober and alerting.
+
+## Prober
+
+[Prober](../prober/) is a go binary being deployed to [cloud run
```

Comment:
Explain what prober does first and then talk about the technical details like cloud run job and cloud scheduler.

Reply:
Added